docker/models/services.py

```diff
@@ -105,6 +105,25 @@ def logs(self, **kwargs):
         )
         return self.client.api.service_logs(self.id, is_tty=is_tty, **kwargs)
 
+    def scale(self, replicas):
+        """
+        Scale service container.
+
+        Args:
+            replicas (int): The number of containers that should be running.
+
+        Returns:
+            ``True``if successful.
+        """
+
+        if not self.attrs['Spec']['Mode'].get('Global'):
```

Comment:
The conditional is wrong.

Reply:
In fact, the conditional was wrong. It returned an empty dictionary and did not raise the exception.